Contents/Sketch/styles.js

```diff
@@ -43,8 +43,7 @@ var applyStyles = function (newStyles, sharedStyles) {
     var existingStyles = sharedStyles.objects();
 
     while(existingStyles.count() > 0) {
-      style = existingStyles.objectAtIndex(0);
-      [existingStyles removeObject:style];
+      [sharedStyles removeSharedObjectAtIndex:0];
```

Comment:
Yup, the color swatches didn't work either. I got the typography working then the colors. Haven't made it to the layer styles yet.